doc/nrf/ug_matter_configuring_env.rst

```diff
@@ -31,12 +31,54 @@ In this setup, Raspberry Pi runs the Thread Border Router, which provides commun
 The controller can be installed on a PC or a mobile phone.
 Both the Thread Border Router and the Matter controller must support IPv6 communication over backbone network, for example Wi-Fi or Ethernet.
 
+.. _ug_matter_configuring_pc_chip_tool:
+
+Configuring Thread Border Router and the CHIP Tool Matter controller on PC
+==========================================================================
+
+In this setup, the Matter controller is installed on PC, and a dedicated Wi-Fi Access Point and the CHIP Tool Matter controller are used.
+This is the recommended setup.
+
+.. matter_env_ctrl_pc_start
+
+.. figure:: images/matter_otbr_controller_separate_pc.svg
+   :width: 600
+   :alt: Setup with OpenThread Border Router and Matter controller on PC
+
+.. matter_env_ctrl_pc_end
+
+Requirements
+------------
+
+To use this setup, you need the following hardware:
+
+* 1x PC with Ubuntu (20.04 or newer)
+* 1x Raspberry Pi Model 3B+ or newer (along with an SD card with at least 8 GB of memory)
+* 1x Wi-Fi Access Point supporting IPv6 (without the IPv6 Router Advertisement Guard enabled on the router)
+* 1x nRF52840 DK or nRF52840 Dongle - for the Radio Co-Processor (RCP) device
+* 1x nRF52840 DK or nRF5340 DK - for the Matter accessory device (programmed with one of :ref:`matter_samples`)
```

Comment:
Maybe we could use a more general term, such as `Nordic Semiconductor's board` as the number of supported board will grow in the future. We already support nRF21 and Thingy:53 (not yet released, but still...).

Reply:
"Nordic Semiconductor's board" could suggest even nRF52811, so it's risky. Is there a way to narrow down the selection? Maybe a requirement that we can mention, like the 1 MB of flash memory? Or link to the Matter HW requirements page once it is ready?

Reply:
Sample README typically list supported boards, so I thought we don't need a further restriction.